controllers/storagecluster/external_resources.go

```diff
@@ -404,6 +404,7 @@ func (r *StorageClusterReconciler) createExternalStorageClusterResources(instanc
 					),
 					isClusterExternal: true,
 				}
+				scc.storageClass.Name = util.GenerateNameForNonResilientCephBlockPoolStorageClass(instance)
```

Comment:
How was the sc creation working so far without a name? Is there something else I am missing